http/headers/Set-Cookie.json

```
@@ -449,32 +443,31 @@
               }
             }
           },
-          "schemeful": {
+          "secure_required_if_none": {
```

Nit: Have you considered calling this `none_requires_secure`?

---

No, but I like your key better.
```suggestion
          "none_requires_secure": {
```